src/app/hooks/inbox-linshare-precomposing-hook.service.js

```diff
@@ -0,0 +1,39 @@
+angular.module('linagora.esn.unifiedinbox.linshare')
+  .factory('inboxLinsharePrecomposingHook', inboxLinsharePrecomposingHook);
```

Comment:
This will cause an issue in Webpack production mode. It should be `inboxLinsharePrecomposingHookFactory`